### DISCUSSION TITLE
A Touchscreen-friendly minimalist UI mode

### DISCUSSION THREAD

**Discussion by SecularSteve** [PrismLauncher/PrismLauncher#2911] [Ideas]
Instead of the standard window popping out when you open the launcher, what if it was something friendlier to tablets, controllers and/or touchscreens?
As an option at least.

I've made a quick wireframe to give you a quick idea of what I mean:

![Prism_TabUI_Wireframe](https://github.com/user-attachments/assets/4820477f-31c6-4789-bb49-1d6cdfa2711c)

**Comment by Trial97** [comment]
https://github.com/PrismLauncher/PrismLauncher/issues/183
